rero_ils/query.py

```diff
@@ -144,9 +147,11 @@ def documents_search_factory(self, search, query_parser=None):
         # organisation public view
         if view != current_app.config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE'):
             org = Organisation.get_record_by_viewcode(view)
-            search = search.filter(
+            filters = Q(
                 'term', holdings__organisation__organisation_pid=org['pid']
             )
+            filters |= Q('exists', field='files')
```

Comment:
If I understand, only document with files are displayed in organisation view right ?
What about other documents without files ?